plugins/module_utils/network/sonic/config/ospfv3/ospfv3.py

```diff
@@ -0,0 +1,551 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The sonic_ospfv3 class
+It is in this file where the current configuration (as dict)
+is compared to the provided configuration (as dict) and the command set
+necessary to bring the current configuration to it's desired end-state is
+created
+"""
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from copy import deepcopy
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.cfg.base import (
+    ConfigBase,
+)
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import (
+    to_list,
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.facts.facts import Facts
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.sonic import (
+    to_request,
+    edit_config
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.utils.utils import (
+    update_states,
+    get_diff,
+    remove_matching_defaults,
+    remove_empties_from_list
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.utils.formatted_diff_utils import (
+    get_new_config,
+    get_formatted_config_diff,
+    __DELETE_CONFIG,
+    __DELETE_LEAFS_OR_CONFIG_IF_NO_NON_KEY_LEAF
+)
+from ansible.module_utils.connection import ConnectionError
+
+PATCH = 'patch'
+DELETE = 'delete'
+DIRECTION = 'IMPORT'
+
+OSPF_PATH = ('/data/openconfig-network-instance:network-instances/network-instance={vrf_name}'
+             '/protocols/protocol=OSPF3,ospfv3/ospfv3')
+
+TEST_KEYS = [
+    {'config': {'vrf_name': ''}},
+    {'helper': {'advertise_router_id': ''}},
+    {'redistribute': {'protocol': ''}}
+]
+
+TEST_KEYS_overridden_diff = [
+    {'config': {'vrf_name': '', '__delete_op': __DELETE_CONFIG}},
+    {'helper': {'advertise_router_id': '', '__delete_op': __DELETE_CONFIG}},
+    {'redistribute': {'protocol': '', '__delete_op': __DELETE_CONFIG}},
+    {'graceful_restart': {'helper': '', '__delete_op': __DELETE_CONFIG}}
+]
+
+TEST_KEYS_diff = [
+    {'config': {'vrf_name': '', '__delete_op': __DELETE_LEAFS_OR_CONFIG_IF_NO_NON_KEY_LEAF}},
+    {'helper': {'advertise_router_id': '', '__delete_op': __DELETE_LEAFS_OR_CONFIG_IF_NO_NON_KEY_LEAF}},
+    {'redistribute': {'protocol': '', '__delete_op': __DELETE_LEAFS_OR_CONFIG_IF_NO_NON_KEY_LEAF}},
+    {'graceful_restart': {'helper': '', '__delete_op': __DELETE_LEAFS_OR_CONFIG_IF_NO_NON_KEY_LEAF}}
+]
+
+DEFAULT_ENTRIES = [
+    #
+]
```

Comment:
Are there any default entries missed out or no default entries?
If no default values, can remove this here and wherever DEFAULT_ENTRIES are being used up.